chacra/models/repos.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+from sqlalchemy import Column, Integer, String, ForeignKey, Boolean, DateTime
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.event import listen
+from sqlalchemy.orm.exc import DetachedInstanceError
+from chacra.models import Base, update_timestamp
+
+
+class Repo(Base):
+
+    __tablename__ = 'repos'
+    id = Column(Integer, primary_key=True)
+    path = Column(String(256))
+    ref = Column(String(256), index=True)
+    distro = Column(String(256), nullable=False, index=True)
+    distro_version = Column(String(256), nullable=False, index=True)
+    modified = Column(DateTime, index=True)
+    signed = Column(Boolean(), default=False)
+    needs_update = Column(Boolean(), default=False)
+    size = Column(Integer, default=0)
+
+    binary_id = Column(Integer, ForeignKey('projects.id'))
```

Comment:
you've called this ``binary_id`` and linked to ``projects.id``. should this be project_id instead?